Stats/src/main/java/io/deephaven/stats/StatsDriver.java

```diff
@@ -150,23 +159,30 @@ public StatsDriver(Clock clock, StatsIntradayLogger intraday, boolean getFdStats
             clockValue = null;
         }
 
-        long now = System.currentTimeMillis();
-        long delay = STEP - (now % STEP);
-        nextInvocation = now + delay;
+        final long now = System.currentTimeMillis();
+        final long delay = STEP - (now % STEP);
+        nextCpuUpdate = now + delay + CPU_INTERVAL;
+        nextMemUpdate = now + delay + MEM_INTERVAL;
```

Comment:
This appears to be a slight change in logic from before - approximately, the previous logic was `now + <X>_INTERVAL`. Assuming this was intentional?

Reply:
It was intentional, yes. Basically, it didn't make sense (to me) that when we next call `update` (fka `timedOut`) and evaluate these, they'll be `delay` behind, and continue to be `delay` behind for the life of the driver.

The whole thing is a little on the silly side, since the intervals match the step. Arguably, I should just remove them and make updatings CPU and memory stats unconditional.